src/chmhtmlwindow.cpp

```diff
@@ -28,12 +28,29 @@
 #include <hhcparser.h>
 #include <memory>
 #include <wx/clipbrd.h>
+#include <wx/dnd.h>
 #include <wx/filename.h>
 #include <wx/log.h>
 #include <wx/uri.h>
 #include <wx/wfstream.h>
 #include <wx/wx.h>
 
+class CHMDropTarget : public wxFileDropTarget {
+    CHMFrame* m_frame;
```

Comment:
Please use `_frame` to match the naming style of the previous code.
Also, please move this to an explicit `private:` section at the end of the class as the current style does.